### DISCUSSION TITLE
Unable to get endpoint /profile

### DISCUSSION THREAD

**Discussion by Exsomnis78** [EDCD/EDMarketConnector#1095] [Q&A]
I am getting "Unable to get endpoint /profile" message at the bottom of EDMC window after ED Odyssey update. Do I need to change anything with the update?

**Comment by Athanasius** [comment]
Check
The
Known
Issues

https://github.com/EDCD/EDMarketConnector/issues/618

Also, this should be fixed in 5.0.2 which will be out today... if people stop distracting me ;) .
